src/zoid/buttons/prerender.jsx

```diff
@@ -31,8 +32,21 @@ export function PrerenderedButtons({ nonce, onRenderCheckout, props } : Prerende
             [ FPTI_KEY.CONTEXT_ID ]:         props.buttonSessionID,
             [ FPTI_KEY.TRANSITION ]:         'process_button_prerender_click'
         }).flush();
-        
-        if (supportsPopups()) {
+
+        if (fundingSource === FUNDING.VENMO) {
+            const target : HTMLElement = event.currentTarget;
+            const spinner = target.querySelector(`.${ CLASS.SPINNER }`);
+            const label = target.querySelector(`.${ CLASS.BUTTON_LABEL }`);
+            if (spinner) {
+                spinner.setAttribute('style', 'display:block !important');
+            }
+            if (label) {
+                label.setAttribute('style', 'display:none;');
+            }
+
+            onRenderCheckout({ fundingSource, card });
+
+        } else if (supportsPopups()) {
             // remember the popup window to prevent showing a new popup window on every click in the prerender state
             if (!win || win.closed) {
```

Comment:
I believe this `window` check needs to happen for all use cases. Otherwise this bug will be introduced: https://github.com/paypal/paypal-checkout-components/pull/1635